Course Material/Introduction to Quantum Computing and FiQCI/00 Superposition and measurement.ipynb

```diff
@@ -0,0 +1,672 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {
+    "qat-category": "1-Circuit generation in Python",
+    "qat-summary": "1-Creation of an EPR pair using two qubits"
+   },
+   "source": [
+    "# Quantum Helloworld: Superposition and measurement\n",
+    "\n",
+    "\n",
+    "The goal of this jupyter notebook is to help you write your first quantum program using Qiskit.\n",
+    "This will help you understand its main functionalities:\n",
+    "\n",
+    "+ Retrieve backend\n",
+    "+ Allocate qubits\n",
+    "+ Apply gates\n",
+    "+ Transpile circuit\n",
+    "+ Submit quantum job\n",
+    "+ Display the results\n",
+    "\n",
+    "In this example, we will build a circuit that creates a quantum superposition state and then run the circuit on Finnish quantum computer, Helmi"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## First step: Retrieve backend\n",
+    "\n",
+    "The first step towards using Qiskit is to import necessary methods:\n",
+    "\n",
+    "+ **qiskit.QuantumCircuit**, **.transpile**, **.execute**: Tools to create, transpile and execute quantum circuits\n",
+    "+ **qiskit.Aer**: Quantum simulator\n",
+    "+ **iqm.qiskit_iqm.IQMProvider**: Used to retrieve instance for Helmi quantum computer\n",
+    "+ **os**: Used to get environment variable for Helmi when using LUMI-supercomputer\n",
+    "\n",
+    "The following cell imports the everything we require in this notebook"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 1,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "from qiskit import QuantumCircuit, transpile, execute\n",
+    "from qiskit import Aer\n",
+    "from iqm.qiskit_iqm import IQMProvider\n",
+    "import os"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "We now have all the tools we need to create the program we want to implement. Now we have to choose backends which we use to run our quantum job\n",
```

Comment:
maybe define that "quantum circuit" == "program" and that a "job" can consists of one or more quantum circuits